### PR TITLE
Added Instana Go exporter to the registry

### PR DESCRIPTION
**Pull request by willianpc**
Update to the content/en/registry with the Instana Go exporter .md file.

### CODE REVIEW

File: content/en/registry/exporter-js-instana.md [hunk 1]
```diff
@@ -12,4 +12,4 @@ license: MIT
 description: The Instana Node.js OpenTelemetry Exporter.
 authors: Instana Authors
 otVersion: latest
----
```
Comment (svrnm) [inline content/en/registry/exporter-js-instana.md:15]: this change seems to be unrelated, please remove

File: content/en/registry/exporter-go-instana.md [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+registryType: exporter
+isThirdParty: true
+language: go
+tags:
+  - Go
+  - exporter
+  - Instana
+repo: https://github.com/instana/go-otel-exporter
+license: MIT
+description: The Instana Node.js OpenTelemetry Exporter.
```
Comment (svrnm) [inline content/en/registry/exporter-go-instana.md:12]: s/Node.js/Go

**Review by svrnm — CHANGES_REQUESTED** [review]

**Comment by willianpc** [comment]
Thanks for the review, @svrnm .
I've applied the requested changes. Could you please re-review it?
TIA,

_ Willian

**Review by svrnm — APPROVED** [review]

**Comment by svrnm** [comment]
lgtm
